viewer/svgoverlay.js

```diff
@@ -19,12 +19,16 @@ export class SvgOverlay {
             padding: 0,
             margin: 0,
             position: "absolute",
+            top: 0,
+            left: 0,
             zIndex: 10000,
             display: "block",
         	"pointer-events": "none"
         });
         
-        document.body.appendChild(svg);
+        const wrapper = document.getElementsByTagName("kros-bim-view")[0];
```

Comment:
mohol by si tu doplniť, že sme to zmenili používame niečo na štýl  // Changed: for section use shift